runtime/src/accounts_db.rs

```diff
@@ -4324,11 +4324,11 @@ impl AccountsDb {
             }
         });
 
-        self.thread_pool_clean.install(|| {
-            unref.into_par_iter().for_each(|key| {
-                self.accounts_index.unref_from_storage(key);
```

Comment:
with this removal, `AccountsIndex::unref_from_storage` is unused.

Reply:
removed it